include/drivers/ipm.h

```diff
@@ -18,6 +18,42 @@
  * @defgroup ipm_interface IPM Interface
  * @ingroup io_interfaces
  * @{
+ *
+ *                      CPU #1        |
+ * +----------+                       |        +----------+
+ * |          +---TX9----+   +--------+--RX8---+          |
+ * |  dev A   |          |   |        |        |  CPU #2  |
+ * |          <---RX8--+ |   | +------+--TX9--->          |
+ * +----------+        | |   | |      |        +----------+
+ *                  +--+-v---v-+--+   |
+ *                  |             |   |
+ *                  |   IPM dev   |   |
+ *                  |             |   |
+ *                  +--+-^---^--+-+   |
+ * +----------+        | |   |  |     |        +----------+
+ * |          <---RX2--+ |   |  +-----+--TX3--->          |
+ * |  dev B   |          |   |        |        |  CPU #3  |
+ * |          +---TX3----+   +--------+--RX2---+          |
+ * +----------+                       |        +----------+
+ *                                    |
+ *
+ * An IPM device is a peripheral capable of passing signals (or data) between
```

Comment:
i wonder is should not instead   `An IPM device is a peripheral capable of passing signals (and data depending on peripherals)`.
means I suppose that a mailbox at least provide signaling  service